sql/updates/world/3.3.5/2016_06_14_99_world.sql

```diff
@@ -11,9 +11,9 @@ UPDATE `creature_template` SET `ScriptName` = '' WHERE `entry` = 23489;
 -- Create SmartAI
 DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid IN (21727, 21725, 22112));
 INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
```

Comment:
SmartAIMgr::LoadSmartAIFromDB: Entry 21725 SourceType 0, Event 1, Link Source Event not found or invalid. Event will never trigger.